src/me/components/DocSearchWidget.scss

```diff
@@ -0,0 +1,178 @@
+$fontcolor: #666666;
+$black: #000;
+$sidebar-search-bg: rgb(70, 70, 82);
+$sidebar-search-shadow: rgba(#0f0e15, 0.5);
+$article-heading: #ffffff;
+$article-hr: #383846;
+$article-text: #d4d7dd;
+$article-link: #00a3ff;
```

Comment:
you're telling me none of these are defined elsewhere?

Reply:
removing